src/markdown-pages/docs/cli-command-reference/simbank-cli-offline.md

```diff
@@ -0,0 +1,121 @@
+---
+path: "/docs/cli-command-reference/simbank-cli-offline"
+title: "Exploring Galasa SimBank offline"
+---
+Distributed with Galasa, SimBank is a component that simulates a mainframe application. It sits above another component called SimPlatform. As delivered, SimBank implements a sample banking application against which you can configure and run a set of provided tests in preparation for running your own tests against an *actual* mainframe application. You can also practice writing some new tests to run against the SimBank banking application.
+
+By exercising the Galasa framework against SimBank, you can pre-empt a lot (but not all) of the work and learning necessary to eventually hook your own tests up with a genuine mainframe environment. If the provided SimBank tests do not work, then it is unlikely that you will be able to run your own tests on a mainframe application. In summary, SimBank helps you to understand Galasa's basic principles of operation before you learn how to connect Galasa to your own mainframe application-under-test.
+
+
+## Launching SimBank offline
+
+SimBank applications and a set of sample SimBank tests are located in the `maven` directory of the `isolated.zip` downloadable file. 
+
+To start exploring the Galasa Simbank application and to run the sample SimBank tests by using the Galasa CLI, you need to complete the following steps:
+
+1. Start the Simplatform server by running the `run-simplatform.sh` script provided in the `isolated.zip`. 
+    1. Navigate to the directory of the zipped distribution that you downloaded, for example, `~/Downloads/isolated`. A `run-simplatform.sh` script is available within the directory. This script starts the Simplatform server which is required to run the SimBank tests.
+    1. Set execute permission on the script by running the `chmod +x run-simplatform.sh` command in the directory containing the `run-simplatform.sh` script. If you are using a Mac, you can set permission to open the Galasa CLI tool by running the `xattr -dr com.apple.quarantine run-simplatform.sh` command in the same directory.
```

Comment:
I don't think this line is needed?
"If you are using a Mac, you can set permission to open the Galasa CLI tool by running the `xattr -dr com.apple.quarantine run-simplatform.sh` command in the same directory."

Reply:
removed